test/proxy/utils/Initializable.test.js

```diff
@@ -184,4 +186,11 @@ contract('Initializable', function (accounts) {
       expect(await this.contract.child()).to.be.bignumber.equal(child);
     });
   });
+
+  describe('disabling initialization', function () {
+    it('old and new patterns dont interfere', async function () {
```

Comment:
I'm not sure this is the right name for this `describe` & `it`. AFAIK, these tests show that they do interfere